core/prisma/seed/seedCommunity.ts

```diff
@@ -1196,3 +1196,5 @@ export const createSeed = <
 	pubs?: PI;
 	forms?: F;
 }) => props;
+
+export type Seed = Parameters<typeof createSeed>[0];
```

Comment:
we can import types without problems and our transactions will work. importing `createSeed` the function, without doing a dynamic import in the test itself, seems to cause the data to persist though

Reply:
smart!